src/Devices/Artemis.Plugins.Devices.Wooting/DataModels/WootingAnalogDataModel.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Artemis.Core.Modules;
+using RGB.NET.Core;
+using System.Collections.Generic;
+
+namespace Artemis.Plugins.Devices.Wooting;
```

Comment:
Namespace doesn't match folder structure, this is the case for most files but I'll add the comment only here